File: frontend/src/app/dto/updateSurveyResponse.ts

export class UpdateSurveyResponse {
  surveyUpdateCount: number
  statusCode: number;
  message: String;
  rowsAffected: number

  constructor(surveyUpdateCount, statusCode, message, rowsAffected) {
    this.surveyUpdateCount = surveyUpdateCount;
    this.statusCode = statusCode;
    this.message = message;
    this.rowsAffected = rowsAffected;
  }
}

File: src/main/java/is/larsen/ebbi/Model/responses/UpdateSurveyResponse.java
package is.larsen.ebbi.Model.responses;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UpdateSurveyResponse extends BaseResponse {
    private Integer rowsAffected;

    public UpdateSurveyResponse(Integer status, String message, Integer rowsAffected) {
        super(status, message);
        this.rowsAffected = rowsAffected;
    }
}
